README.md

```diff
@@ -55,6 +55,23 @@ For general questions about OCI, please see the [FAQ on the OCI site](https://ww
 
 The [GitHub milestones](https://github.com/opencontainers/distribution-spec/milestones) lay out the path to the future improvements.
 
+# Extensions
+
+The Distribution Specification Project is including, herein, a process and API for prototyping
+and testing extensions to the Distribution API.
+
+We invite contributions, comments, and reviews to these extensions.
+These extensions will only move forward with significant support from registies, registry clients, and users.
+
+Disclaimer: Unless stated otherwise, extension prototypes are subject to change. Extensions
+should NOT be considered as final or approved official extensions to the OCI Distribution API until they
+have been included in an OCI Distribution Specification.
```

Comment:
It seems like this is the case now. Should we close this PR @mikebrow ?

Reply:
I need to update this PR to point to https://github.com/opencontainers/distribution-spec/blob/main/extensions/README.md